android/src/com/mapswithme/util/bottomsheet/MenuBottomSheetItem.java

```diff
@@ -0,0 +1,42 @@
+package com.mapswithme.util.bottomsheet;
+
+public class MenuBottomSheetItem
+{
+    private final int id;
```

Comment:
Why not make them public and remove getters? `final` variables can not be modified, and the code will be cleaner. It works as a simple `struct` container for several variables.